src/components/Task/style.scss

```diff
@@ -0,0 +1,54 @@
+.todo-list {
+  &__task-container {
+    display: flex;
+    align-items: center;
+  }
+
+  &__task-container button {
+    margin-left: 10px;
+  }
+
+  &__task-container img:hover {
+    cursor: pointer;
+  }
+
+  &__text-task {
+    font-size: 18px;
+    color: white;
+    margin-left: 8px;
+  }
+
+  &__done-text {
+    color: rgba(255, 255, 255, 1);
+    text-decoration: line-through;
+  }
+
+  &-button {
+    display: flex;
+    flex-direction: row;
+    border: 0px;
+    height: 24px;
+    width: 24px;
+    background: none;
+  }
+
+  &__hide {
+    display: none;
+  }
+}
+
+.container {
+  &__unchecked {
```

Comment:
make the most of nesting. __unchecked can be a modifier. That is
task-container__unchecked